src/MostUsed/Parser/Common.hs

```diff
@@ -0,0 +1,48 @@
+module MostUsed.Parser.Common
+    ( item
+    , pipe
+    ) where
+
+import Data.Char (isSpace, isPrint)
+import MostUsed.Types
+import Text.Megaparsec
+import Text.Megaparsec.String
+
+item :: Parser Item
+item = do
+    command <- some allowedCharsInBareWords
+    space
+    arguments <- singleArgument `sepEndBy` separator
+    return $ Item command arguments
+
+pipe :: Parser ()
+pipe = space >> char '|' >> space
```

Comment:
Should the spaces be optional? You could in theory do like `cat foo.txt|xargs echo` and have that be a valid command.